cygnus-common/src/main/java/com/telefonica/iot/cygnus/nodes/CygnusApplication.java

```diff
@@ -162,6 +159,14 @@ public synchronized void handleConfigurationEvent(MaterializedConfiguration conf
      */
     public static void main(String[] args) {
         try {
+            // Set some MDC logging fields to 'N/A' for this thread
+            // Later in this method the component field will be given a value
+            org.apache.log4j.MDC.put(CommonConstants.LOG4J_CORR, "N/A");
```

Comment:
I'd suggest to use a constant for "N/A". I understand it should be global, as it is used in several files.